.internal-ci/helm/fog-ledger/values.yaml

```diff
@@ -0,0 +1,234 @@
+imagePullSecrets:
+- name: docker-credentials
+
+# Pods share the image tag.
+image:
+  org: mobilecoin
+  tag: '' # Overrides the image tag whose default is the chart appVersion.
+
+# Mobilecoin network instance
+mobilecoin:
+  network: ''
+  partner: ''
+
+fogLedger:
+  # Stack configurations by network.
+  stackConfig:
+    network:
+      # Assume default is a dev network. We can always define a "network" value if needed.
+      default:
+        shardSize: 20_000
+        exceedBlockHeightBy: 5_000
+        shardOverlap: 0
+        count: 2
+        blockHeightRetrieval:
+          blockCountURL: 'https://node1.{{ .Release.Namespace }}.development.mobilecoin.com/gw/consensus_common.BlockchainAPI/GetLastBlockInfo'
+          responseJQ: '.index'
+          queryInterval: 1m
+          requestBody: ''
+      test:
+        shardSize: 400_000
+        exceedBlockHeightBy: 10_000
+        shardOverlap: 0
+        count: 2
+        blockHeightRetrieval:
+          blockCountURL: https://node1.test.mobilecoin.com/gw/consensus_common.BlockchainAPI/GetLastBlockInfo
+          responseJQ: '.index'
+          queryInterval: 5m
+          requestBody: ''
+      main:
+        shardSize: 400_000
+        exceedBlockHeightBy: 10_000
+        shardOverlap: 0
+        count: 3
+        blockHeightRetrieval:
+          blockCountURL: https://node1.prod.mobilecoinww.com/gw/consensus_common.BlockchainAPI/GetLastBlockInfo
+          responseJQ: '.index'
+          queryInterval: 5m
+          requestBody: ''
+
+  router:
+    ### list of fog-ledger-router hostnames (client responder ID)
+    hosts:
+    - partner: ''
+      responderID: ''
+
+    replicaCount: 1
+
+    image:
+      org: ''
+      name: fog-ledger
+      pullPolicy: Always
+
+    resources:
+      limits:
+        intel.com/sgx: 5000
+        memory: 5Gi
+      requests:
+        intel.com/sgx: 5000
+        memory: 5Gi
+        cpu: 1100m
+
+    nodeSelector:
+      sgx-enabled-node: 'true'
+
+    tolerations:
+    - key: sgx
+      operator: Equal
+      value: 'true'
+      effect: NoSchedule
+
+    # disable affinity rules for single node testing
+    podManagementPolicy: Parallel
+    affinityEnabled: true
+    topologySpreadConstraintsEnabled: true
```

Comment:
I may be misunderstanding the comment or this setting, but it seems like afinity rules are being enabled here.

Reply:
Affinity rules are enabled by default.

We may want to disabled them if you wanted to use the helm chart for small local cluster and single node testing.